packages/vscode-extension/src/webview/components/shared/KyebindingInfo.tsx

```diff
@@ -0,0 +1,59 @@
+import { ReactNode, useEffect, useState } from "react";
+import "./KeybindingInfo.css";
+import { useProject } from "../../providers/ProjectProvider";
+
+interface KeybindingInfoProps {
+  commandName: string;
+}
+
+interface SymbolProps {
+  children: ReactNode;
+}
+
+const Symbol = ({ children }: SymbolProps) => {
+  return <span className="symbol">{children}</span>;
+};
```

Comment:
This doesn't seem like a particularily interesting component, you can just inline <span/> in the place where you use it